lib/pbench/test/functional/agent/cli/commands/triggers/test_register_tool_trigger.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+
+
+def test_pbench_register_tool_trigger_help():
```

Comment:
Again, doc strings to describe each test case would be nice.

While combining multiple test cases into one method like your `test_register_triggers` is convenient to leverage common setup, you could also do that through a fixture and keep the test cases separate for easier reading and triage when something goes wrong.